src/plugins/data/common/index_patterns/types.ts

```diff
@@ -185,6 +185,15 @@ export interface FieldSpec {
   readFromDocValues?: boolean;
   subType?: IFieldSubType;
   indexed?: boolean;
+
+  /**
+   * @experimental These fields are experimental and are subject to change
+   * TODO: Refactor and move them into DataViewField
+   */
+  suggestions?: {
+    autoCompleteValues?: string[];
```

Comment:
nit: one more pitch if this could be just `values`. autocomplete is one user but it doesn't have to be the only user of this new field